src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -202,6 +204,24 @@ public function stream($view, array $parameters = array(), StreamedResponse $res
     }
 
     /**
+     * Returns a HttpException.
+     *
+     * This will result in a response based on given status code. Usage example:
+     *
+     *     throw $this->createHttpException(500);
+     *
+     * @param int             $statusCode The HTTP status code
+     * @param string          $message    A message
+     * @param \Exception|null $previous   The previous exception
+     *
+     * @return HttpException
+     */
+    public function createHttpException($statusCode, $message = null, \Exception $previous = null)
```

Comment:
@hhamon, I think it would be great, but again: I've tried to make it similar to pre-existing exception shortcuts. If they are still public only for BC reasons, I'll change the visibility ASAP; but I before need to know some thoughts about it.